frontend/staff-dashboard/src/styles/antd.less

```diff
@@ -8,11 +8,7 @@
 @primary-color: #A31F34;
 
 .income-usd {
-  max-width: 100px;
-  overflow-x: scroll;
-  height: 35px;
-
-  span {
-    white-space: nowrap;
-  }
+  max-width: 150px;
+  overflow-y: scroll;
```

Comment:
`overflow-y: scroll` here makes the bar visible if you have scrollbars always visible (on macOS) turned on:
![image](https://user-images.githubusercontent.com/945611/184636869-21d6f32b-4a80-4c5a-b21e-58f140d24d55.png)

Should be `auto` instead.